js/screenSnippet/ScreenSnippet.js

```diff
@@ -74,6 +74,7 @@ class ScreenSnippet {
             child = childProcess.execFile(captureUtil, captureUtilArgs, (error) => {
                 // will be called when child process exits.
                 if (error && error.killed) {
+                    log.send(logLevels.ERROR, 'ScreenSnippet: ERROR executing command: '+ error);
```

Comment:
don't think needed this is legitimate, as we kill any previous process. so only one process at a time.